uamqp/management_operation.py

```diff
@@ -45,37 +46,63 @@ def _on_amqp_management_open_complete(self, result):
     def _on_amqp_management_error(self):
         """Callback run if an error occurs in the send/receive links."""
         # TODO: This probably shouldn't be ValueError
-        self.mgmt_error = ValueError("Management Operation error ocurred.")
+        self.mgmt_error = ValueError("Management Operation error occurred.")
```

Comment:
Should this attribute be private?

Reply:
I have updated the implementation to add a ready() method to `ManagementExecu so that we could make both mgmt_error and mgmt_link_open_status be private